src/Common/ErrorCodes.cpp

```diff
@@ -1,526 +1,562 @@
+    M(INVALID_TEMPLATE_FORMAT, 474) \
+    M(INVALID_WITH_FILL_EXPRESSION, 475) \
+    M(WITH_TIES_WITHOUT_ORDER_BY, 476) \
+    M(INVALID_USAGE_OF_INPUT, 477) \
+    M(UNKNOWN_POLICY, 478) \
+    M(UNKNOWN_DISK, 479) \
+    M(UNKNOWN_PROTOCOL, 480) \
+    M(PATH_ACCESS_DENIED, 481) \
+    M(DICTIONARY_ACCESS_DENIED, 482) \
+    M(TOO_MANY_REDIRECTS, 483) \
+    M(INTERNAL_REDIS_ERROR, 484) \
+    M(SCALAR_ALREADY_EXISTS, 485) \
+    M(CANNOT_GET_CREATE_DICTIONARY_QUERY, 487) \
+    M(UNKNOWN_DICTIONARY, 488) \
+    M(INCORRECT_DICTIONARY_DEFINITION, 489) \
+    M(CANNOT_FORMAT_DATETIME, 490) \
+    M(UNACCEPTABLE_URL, 491) \
+    M(ACCESS_ENTITY_NOT_FOUND, 492) \
+    M(ACCESS_ENTITY_ALREADY_EXISTS, 493) \
+    M(ACCESS_ENTITY_FOUND_DUPLICATES, 494) \
+    M(ACCESS_STORAGE_READONLY, 495) \
+    M(QUOTA_REQUIRES_CLIENT_KEY, 496) \
+    M(ACCESS_DENIED, 497) \
+    M(LIMIT_BY_WITH_TIES_IS_NOT_SUPPORTED, 498) \
+    M(S3_ERROR, 499) \
+    M(CANNOT_CREATE_DATABASE, 501) \
+    M(CANNOT_SIGQUEUE, 502) \
+    M(AGGREGATE_FUNCTION_THROW, 503) \
+    M(FILE_ALREADY_EXISTS, 504) \
+    M(CANNOT_DELETE_DIRECTORY, 505) \
+    M(UNEXPECTED_ERROR_CODE, 506) \
+    M(UNABLE_TO_SKIP_UNUSED_SHARDS, 507) \
+    M(UNKNOWN_ACCESS_TYPE, 508) \
+    M(INVALID_GRANT, 509) \
+    M(CACHE_DICTIONARY_UPDATE_FAIL, 510) \
+    M(UNKNOWN_ROLE, 511) \
+    M(SET_NON_GRANTED_ROLE, 512) \
+    M(UNKNOWN_PART_TYPE, 513) \
+    M(ACCESS_STORAGE_FOR_INSERTION_NOT_FOUND, 514) \
+    M(INCORRECT_ACCESS_ENTITY_DEFINITION, 515) \
+    M(AUTHENTICATION_FAILED, 516) \
+    M(CANNOT_ASSIGN_ALTER, 517) \
+    M(CANNOT_COMMIT_OFFSET, 518) \
+    M(NO_REMOTE_SHARD_AVAILABLE, 519) \
+    M(CANNOT_DETACH_DICTIONARY_AS_TABLE, 520) \
+    M(ATOMIC_RENAME_FAIL, 521) \
+    M(UNKNOWN_ROW_POLICY, 523) \
+    M(ALTER_OF_COLUMN_IS_FORBIDDEN, 524) \
+    M(INCORRECT_DISK_INDEX, 525) \
+    M(UNKNOWN_VOLUME_TYPE, 526) \
+    M(NO_SUITABLE_FUNCTION_IMPLEMENTATION, 527) \
+    M(CASSANDRA_INTERNAL_ERROR, 528) \
+    M(NOT_A_LEADER, 529) \
+    M(CANNOT_CONNECT_RABBITMQ, 530) \
+    M(CANNOT_FSTAT, 531) \
+    M(LDAP_ERROR, 532) \
+    M(INCONSISTENT_RESERVATIONS, 533) \
+    M(NO_RESERVATIONS_PROVIDED, 534) \
+    M(UNKNOWN_RAID_TYPE, 535) \
+    M(CANNOT_RESTORE_FROM_FIELD_DUMP, 536) \
+    M(ILLEGAL_MYSQL_VARIABLE, 537) \
+    M(MYSQL_SYNTAX_ERROR, 538) \
+    M(CANNOT_BIND_RABBITMQ_EXCHANGE, 539) \
+    M(CANNOT_DECLARE_RABBITMQ_EXCHANGE, 540) \
+    M(CANNOT_CREATE_RABBITMQ_QUEUE_BINDING, 541) \
+    M(CANNOT_REMOVE_RABBITMQ_EXCHANGE, 542) \
+    M(UNKNOWN_MYSQL_DATATYPES_SUPPORT_LEVEL, 543) \
+    M(ROW_AND_ROWS_TOGETHER, 544) \
+    M(FIRST_AND_NEXT_TOGETHER, 545) \
+    M(NO_ROW_DELIMITER, 546) \
+    M(INVALID_RAID_TYPE, 547) \
+    M(UNKNOWN_VOLUME, 548) \
+    \
+    M(KEEPER_EXCEPTION, 999) \
+    M(POCO_EXCEPTION, 1000) \
+    M(STD_EXCEPTION, 1001) \
+    M(UNKNOWN_EXCEPTION, 1002) \
+    \
+    M(CONDITIONAL_TREE_PARENT_NOT_FOUND, 2001) \
+    M(ILLEGAL_PROJECTION_MANIPULATOR, 2002) \
```

Comment:
BTW, if we change from 
UNKNOWN_VOLUME, 548
to
548, UNKNOWN_VOLUME
the code will be slightly more readable.